Setup/docker-compose_dev.yml

```diff
@@ -374,8 +363,9 @@ services:
         networks:
           - shield
         environment:
-            - "SHIELD_IP=IP_ADDRESS"
+            - "SHIELD_IP=192.168.56.110"
```

Comment:
Please replace with IP_ADDRESS

Reply:
Reverted everything in https://github.com/EricomSoftwareLtd/Shield/pull/47/commits/8398ad3972036beca9073f52f7c444e2fd7a5184